system/Router/RouteCollectionInterface.php

```diff
@@ -30,10 +30,8 @@ interface RouteCollectionInterface
      *
      * @param array|Closure|string $to
      * @param array                $options
-     *
-     * @return mixed
      */
-    public function add(string $from, $to, ?array $options = null);
+    public function add(string $from, $to, ?array $options = null): self;
```

Comment:
This is a breaking change.
https://3v4l.org/n9AYI

Reply:
I have been removed.